runtime/sema/type.go

```diff
@@ -4982,15 +4968,22 @@ func (t *AddressType) GetMembers() map[string]MemberResolver {
 // However, to check if a type *strictly* belongs to a certain category, then consider
 // using `IsSameTypeKind` method. e.g: "Is type `T` an Integer type?". Using this method
 // for the later use-case may produce incorrect results.
-//   * IsSubType()      - To check the assignability. e.g: Is argument type T is a sub-type
-//                        of parameter type R. This is the more frequent use-case.
-//   * IsSameTypeKind() - To check if a type strictly belongs to a certain category. e.g: Is the
-//                        expression type T is any of the integer types, but nothing else.
-//                        Another way to check is, asking the question of "if the subType is Never,
-//                        should the check still pass?". A common code-smell for potential incorrect
-//                        usage is, using IsSubType() method with a constant/pre-defined superType.
-//                        e.g: IsSubType(<<someType>>, FixedPointType)
 //
+// The differences between these methods is as follows:
+//
+//	IsSubType()
+//
+// To check the assignability, e.g: is argument type T is a sub-type
+// of parameter type R? This is the more frequent use-case.
+//
+//	IsSameTypeKind()
+//
+// To check if a type strictly belongs to a certain category. e.g: Is the
+// expression type T is any of the integer types, but nothing else.
+// Another way to check is, asking the question of "if the subType is Never,
+// should the check still pass?". A common code-smell for potential incorrect
+// usage is, using IsSubType() method with a constant/pre-defined superType.
+// e.g: IsSubType(<<someType>>, FixedPointType)
```

Comment:
The default automatic formatting for this one seems ok to me:
```suggestion
//   - IsSubType()      - To check the assignability. e.g: Is argument type T is a sub-type
//     of parameter type R. This is the more frequent use-case.
//   - IsSameTypeKind() - To check if a type strictly belongs to a certain category. e.g: Is the
//     expression type T is any of the integer types, but nothing else.
//     Another way to check is, asking the question of "if the subType is Never,
//     should the check still pass?". A common code-smell for potential incorrect
//     usage is, using IsSubType() method with a constant/pre-defined superType.
//     e.g: IsSubType(<<someType>>, FixedPointType)
```

Reply:
@turbolent thoughts? this was one example you specifically cited as being broken in a previous PR formatted with gofmt 1.19

Reply:
Ah, I missed this thread while reviewing, but had also commented on it here: https://github.com/onflow/cadence/pull/2041#discussion_r989524751